### PR TITLE
 User can see their account page with listing & content with received bids on them

### PR DESCRIPTION
**Pull request by Felix-Willborg**
### User Story
```
As a user
In order to be able to know what listings i have posted
I would like to have an My Account page with listings details and related bids on them
```
## Changes proposed in this pull request:
* adding My Account page where user can see their listings and incoming bids
## What I have learned working on this feature:
* practice Get requests using axios 
## Screenshots (Client)
Skärmavbild 2020-09-08 kl  11 35 38
Skärmavbild 2020-09-08 kl  15 30 58

### CODE REVIEW

File: cypress/integration/userCanSeeAccountDetails.feature.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    cy.get("[data-cy=button]").contains("My Account").click();
+    cy.get("[data-cy=listing-1]").within(() => {
+      cy.get("[data-cy=lead]").should("contain", "Vacant");
+      cy.get("[data-cy=scene]").should("contain", "indoor");
+      cy.get("[data-cy=image]").should("exist");
+      cy.get("[data-cy=category]").should("contain", "Parking spot");
+    });
+  });
+
+  it("user can see his/hers listings", () => {
```
Comment (Alex-Bellamy) [inline cypress/integration/userCanSeeAccountDetails.feature.js]: perhaps say user can see listing two (like above it block)?

File: cypress/fixtures/account_show.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "listing": {
+    "id": 2,
+    "lead": "Great parking spot in central of Stockholm.",
+    "scene": "indoor",
+    "category": "Parking spot",
+    "images": ["https://i.redd.it/w3kr4m2fi3111.png"],
+    "description": "Heated garage in the middle of Stockholm that fits one big SUV.",
+    "address": "Sibyllegatan 18, 11442 Stockholm",
+    "price": 200,
+    "bidding": [
```
Comment (viktorsommar) [inline cypress/fixtures/account_show.json]: We have it as "biddings" in the api

File: cypress/integration/userCanSeeAccountDetails.feature.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
+describe("User can see account page", () => {
```
Comment (viktorsommar) [inline cypress/integration/userCanSeeAccountDetails.feature.js:1]: What happens if a user goes to the url without being logged in?

**Review by Alex-Bellamy — CHANGES_REQUESTED** [review]
sad path?

File: src/components/MyOwnListing.jsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  }, []);
+
+  const getMySingleListing = async () => {
+    const headers = JSON.parse(localStorage.getItem("J-tockAuth-Storage"));
+    let response = await axios.get(`account/listings/${id}`, {
+      headers: headers,
+    });
+    setMySingleListing(response.data.listing);
+    setImages(response.data.listing.images);
+    setBiddings(response.data.listing.bidding);
```
Comment (viktorsommar) [inline src/components/MyOwnListing.jsx]: We have it as biddings in the api

**Review by viktorsommar — CHANGES_REQUESTED** [review]

**Comment by Alex-Bellamy** [comment]
semaphore also fails

File: cypress/integration/userCanSeeAccountDetails.feature.js (hunk 3 above)
Comment (Felix-Willborg) [reply cypress/integration/userCanSeeAccountDetails.feature.js:1]: Nothing shows. That is restricted in the backend.

**Review by viktorsommar — APPROVED** [review]

**Review by Alex-Bellamy — APPROVED** [review]

**Pull request merged**